aiohttp_session/redis_storage.py

```diff
@@ -43,13 +43,14 @@ def __init__(
             decoder=decoder,
         )
         if aioredis is None:
-            raise RuntimeError("Please install aioredis")
+            raise RuntimeError("Please install redis")
         # May have installed aioredis separately (without aiohttp-session[aioredis]).
-        if StrictVersion(aioredis.__version__).version < (2, 0):
-            raise RuntimeError("aioredis<2.0 is not supported")
+        lib_version = tuple(map(int, VERSION[:2]))
```

Comment:
This is either already an int, or this code will error. So, if we're confident they are ints, we should just do a `cast()`.